### PR TITLE
Adding missing functions

### LINKED ISSUE #222: Functions missing from documentation
A number of functions have not been added to the documentation. e.g. `kcorrect_spectra` and `schechter_smf` are both missing from the "Galaxies" documentation page (https://readthedocs.org/projects/skypy/builds/11457398/). Somebody should go through all of the public functions to check they have an entry in the documentation and add any that are missing.

### PR DESCRIPTION
**Pull request by Lucia-Fonseca**
## Description
This pull request adds missing functions in the documentation. It resolves #222.
## Checklist
- [x] Follow the [Contributor Guidelines](https://github.com/skypyproject/skypy/blob/master/CONTRIBUTING.md)
- [ ] Write unit tests
- [ ] Write documentation strings
- [ ] Assign someone from your working team to review this pull request
- [ ] Assign someone from the infrastructure team to review this pull request

### CODE REVIEW

**Comment by Lucia-Fonseca** [comment]
Ready now.

File: docs/galaxy.rst [hunk 1]
```diff
@@ -48,6 +48,7 @@ The following models are found in the `skypy.galaxy.redshift` package.
 .. autosummary::
    :nosignatures:
 
+   herbel_pdf
```
Comment (rrjbca) [inline docs/galaxy.rst:51]:
```suggestion
```
`herbel_pdf` is only used internally in `herbel_redshift` and so should not be added to the documentation.

File: docs/halo/index.rst [hunk 2]
```diff
@@ -15,3 +15,9 @@ Mass (`skypy.halo.mass`)
 ========================
 
 .. automodule:: skypy.halo.mass
+
+
+Mass (`skypy.halo.quenching`)
```
Comment (rrjbca) [inline docs/halo/index.rst:20]:
```suggestion
Quenching (`skypy.halo.quenching`)
```

File: skypy/galaxy/redshift.py [hunk 3]
```diff
@@ -12,6 +12,7 @@
 
 
 __all__ = [
+    'herbel_pdf',
```
Comment (rrjbca) [inline skypy/galaxy/redshift.py:15]:
```suggestion
```
Again don't add `herbel_pdf` here

**Comment by Lucia-Fonseca** [comment]
Fixed

**Pull request merged**